packages/server-wallet/src/models/channel.ts

```diff
@@ -426,7 +435,11 @@ export class Channel extends Model implements RequiredColumns {
     for (const signedState of this.sortedStates) {
       // If there is not a valid transition we know there cannot be a valid support
       // so we clear out what we have and start at the current signed state
-      if (previousState && !this.validChain(signedState, previousState)) {
+
+      if (
+        previousState &&
+        !validateTransition(signedState, previousState, logger, byteCode, skipAppTransition)
```

Comment:
Would you expect this change to affect performance?

Reply:
I would not, no. The new function behaves very similarly to the old one when either `skipAppTransition=true` or `skipAppTransition=false && byteCode=undefined`.

Reply:
In other words, the app transition check will fail early for null apps, without hitting pure evm.